### DISCUSSION TITLE
Documentazione e wiki

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#23] [Announcements]
Siccome sto perdendo il filo di quali sono gli episodi completati ho scritto una paginetta wiki in cui è possibile trovare lo stato di tutti gli episodi:

https://github.com/pdonadeo/e-uscito-joypad/wiki/Stato-della-compilazione-dei-dati

Fatemi un favore: potete dirmi nuovamente e da capo quali episodi sono stati completati? In questo modo posso completare la pagina di wiki, che ci serve anche per sapere cosa fare e cosa è stato fatto.

**Comment by agofa** [comment]
Io ho fatto ep2, ep3, ep4 e ep5

**Reply by pdonadeo** [reply]
Ma puoi mettere la spunta o devo farlo io? Mi sa che posso editore solo io.

**Reply by agofa** [reply]
Stavo per scrivere che non è modificabile, almeno non ci sono riuscito

**Comment by agofa** [comment]
Dovrebbero essere pronti anche 1, 7 e 8
Non so il 6

**Reply by n1k9** [reply]
si io ho fatto 7 e 8

**Comment by pdonadeo** [comment]
@agofa grazie: ho segnato 1, 2, 3, 4, 5, 7 e 8 come completati.

**Comment by bionicco** [comment]
I video sono tutti fatti.
PS: Però ora abbiamo 2 thread e una wiki con gli episodi fatti/in lavorazione. Forse almeno un thread andrebbe chiuso

**Reply by pdonadeo** [reply]
Usiamo questo thread, per favore, per segnalare che gli episodi sono fatti. Il wiki lo aggiorno io leggendo da questo thread.

Ora aggiorno il wiki, tutti i video risultano fatti 😃

**Reply by agofa** [reply]
Quando si prende in carico un episodio lo si potrebbe scrivere nelle note.
Almeno io farò così

**Comment by agofa** [comment]
Fatto ep6

**Reply by pdonadeo** [reply]
Grazie @agofa, segnato nel wiki.

**Comment by pdonadeo** [comment]
Inizio l'episodio 9 ["quello con Final Fantasy VII Remake e gli altri"](https://db.euscitojoypad.it/backoffice/episodio/14/change/).

Quando lo finisco lo scrivo e aggiorno il wiki.

**Comment by bionicco** [comment]
vado sul 10

**Reply by bionicco** [reply]
Fatto!
(pochi titoli)
PS: per ascoltare ho usato google podcast che permette di regolare la velocità di riproduzione, lo consiglio

**Reply by agofa** [reply]
> PS: per ascoltare ho usato google podcast che permette di regolare la velocità di riproduzione, lo consiglio

Grazie! Ottimo consiglio

**Comment by pdonadeo** [comment]
Perfetto: segno 9 e 10 finiti.

**Comment by agofa** [comment]
Prendo 11, ma non trattenete il respiro nell'attesa ...

**Reply by agofa** [reply]
Fatto 11 con commenti
Un po' di indeterminatezza sui giochi di tennis, hanno citato giochi dal nome simile in modo un po' caotico, ne ho messi un po' spero nell'ordine giusto
Al minuto 40:00 Zampa consiglia un libro e non un gioco, Sega Arcade Pop-up History (Read Only Memory), che non so come inserire

**Comment by pdonadeo** [comment]
Episodio 11 segnato completo.

Inizio episodio 12.

**Comment by pdonadeo** [comment]
Finito episodio 12.

**Comment by MarcoJul** [comment]
Chiedo un favore LATO frontend (per chi é in pari con i Tienimi Bordone)

Se avete possibilità, potreste completare il database di un paio delle puntate piū recenti? Giusto perché cosī in fase di mio test riesco a vedere subito gli effetti senza dover scrollare in giro per la lista.

Richiesta da pigro, ma se c'é la possibilità di farlo senza stravolgere troppo i programmi mi faciliterebbe il lavoro!

**Reply by agofa** [reply]
OK allora prendo il 48, inizio ora

**Reply by agofa** [reply]
Fatto 48

Si parla di noi al minuto 35:45 (Paolo) ....

**Reply by pdonadeo** [reply]
> Fatto 48

Segnato nel wiki, grazie.

**Reply by pdonadeo** [reply]
@MarcoJul ho fatto una esportazione del JSON che contiene anche l'episodio 48 appena fatto da @agofa.

Ora non posso fare un deploy e comunque aspetto che tu mi dia il via così faccio tutto una volta sola.

Te lo allego qui così lo puoi usare sul tuo PC.

[db_data.json.txt](https://github.com/pdonadeo/e-uscito-joypad/files/8791393/db_data.json.txt)

Devi rinominarlo, GitHub non mi fa allegare file JSON.

**Reply by MarcoJul** [reply]
Ok grazie mille, per oggi comunque a breve devo staccarmi dal computer altrimenti rischio l'incidente diplomatico😅

Quando stacco vi aggiorno su quanto aspettarsi di aggiornato.

**Comment by agofa** [comment]
Prendo il 13-1

**Reply by agofa** [reply]
Fatto 13-1 che finisce troncato perché prosegue in 13-2
Credo sia diviso in due solo sul sito del post
Su Google podcast e spotify (immagino quindi ovunque) è un solo blocco
@pdonadeo lo lasciamo diviso in due come da sito post?

**Reply by pdonadeo** [reply]
Secondo me sì, manterrei il Post come fonte editoriale reale: le piattaforme fanno un po' quello che vogliono.

**Reply by agofa** [reply]
Va bene, prendo 13-2

**Reply by agofa** [reply]
Fatto anche 13-2
Al minuto 22:30 parlano di Project Athia ma non l'ho trovato, ho messo nota

**Reply by agofa** [reply]
> Secondo me sì, manterrei il Post come fonte editoriale reale: le piattaforme fanno un po' quello che vogliono.

Forse il sito del post, il player, gestisce male file audio troppo lunghi

**Reply by pdonadeo** [reply]
Non credo, pensa a Morning (30 minuti) o aild alcuni episodi delk'Eurovision (oltre un'ora)

**Reply by agofa** [reply]
Vero. Allora, boh?

**Reply by pdonadeo** [reply]
@agofa il 13-2 è finito?

**Reply by pdonadeo** [reply]
Non è per mettere fretta, solo per completezza, così segno nel wiki.

**Reply by agofa** [reply]
> Non è per mettere fretta, solo per completezza, così segno nel wiki.

Ehm sì, scritto un po' sopra

**Reply by agofa** [reply]
> Fatto anche 13-2
> Al minuto 22:30 parlano di Project Athia ma non l'ho trovato, ho messo nota

Qui

**Reply by MarcoJul** [reply]
> Fatto anche 13-2
> 
> Al minuto 22:30 parlano di Project Athia ma non l'ho trovato, ho messo nota

Project Athia se non sbaglio é diventato Forspoken nel mentre, magari quello è presente

**Reply by agofa** [reply]
> Forspoken

Credo tu abbia ragione, più tardi farò verifica e aggiornerò

**Reply by pdonadeo** [reply]
> > Fatto anche 13-2
> > Al minuto 22:30 parlano di Project Athia ma non l'ho trovato, ho messo nota
> 
> Qui

Segnato.

**Reply by agofa** [reply]
> Project Athia se non sbaglio é diventato Forspoken nel mentre, magari quello è presente

Trovato e episodio aggiornato
Grazie @MarcoJul

**Comment by agofa** [comment]
Inizio episodio 47, pro test front end

**Reply by agofa** [reply]
> Inizio episodio 47, pro test front end

Fatto ep 47
Al minuto 10:10 Fossa cita un gioco Lego che sembra chiamarsi Brick Tales o Bricktales ma non sono sicuro, non lo trovo
Usando gogol ho trovato un bricktales, ma non c'è su rawg
https://store.steampowered.com/app/1898290/LEGO_Bricktales/

**Reply by pdonadeo** [reply]
Ho segnato come completato. Poi vedrò di aggiungere a mano il gioco.

**Comment by agofa** [comment]
Inizio il 46

**Reply by agofa** [reply]
> Inizio il 46

Finito 46, @pdonadeo 
Al minuto 50:36 si parla per la prima volta di noi: qualcuno farà euscitojoypad?  :-)

**Reply by pdonadeo** [reply]
46 segnato, grazie.

**Comment by n1k9** [comment]
ho finito la 14 (episodio!!!)

**Reply by pdonadeo** [reply]
Segnato, grazie!

**Comment by agofa** [comment]
Inizio il 49

**Reply by agofa** [reply]
> Inizio il 49

Ep 49 fatto con commenti
Minuto 19:12 Zampa parla di Gunbrella che non trovo
(https://www.nintendo.com/store/products/gunbrella-switch/)

41:30 Zampa consiglia un libro, The King of Fighters: The Ultimate History

**Reply by MarcoJul** [reply]
Credo che sia in uscita nel 23, magari non c'è ancora la scheda🫢

**Reply by pdonadeo** [reply]
> Ep 49 fatto con commenti

Segnato. Possiamo tranquillamente affermare che Agostino sia una autentica **macchina da guerra**!

**Reply by agofa** [reply]
> Credo che sia in uscita nel 23, magari non c'è ancora la scheda

Infatti
A volte ci sono schede di giochi con data uscita TBD, ma non per tutti

**Comment by agofa** [comment]
E vabbé, cerco di farne 1 o 2 a sera, ma nel luuungo weekend che sta per arrivare potrò far poco

**Comment by sanvisimo** [comment]
Ciao, io sono arrivato ora, se serve posso dare una mano e inserire episodi. Nella vita faccio anche sviluppo web, quindi me la cavo con react, se dovesse servire una mano, mi sono un po' perso nelle discussioni e non so bene a chi chiedere

Grazie

**Reply by pdonadeo** [reply]
@sanvisimo
Ciao Simone, benvenuto a bordo! Allo stato attuale siamo abbastanza coperti _lato coding_ (cit.) mentre quello che serve davvero è riascoltare gli episodi mancanti di Joypad ed introdurre nel [database](https://db.euscitojoypad.it/) i giochi, il minuto in cui vengono citati, e qualche altra informazione. Per fare questo ti serve accesso al database: puoi scrivermi privatamente a paolo chiocciola donadeo punto net e ti faccio un account.

L'elenco degli episodi già completati si trova su [questa pagina dei wiki](https://github.com/pdonadeo/e-uscito-joypad/wiki/Stato-della-compilazione-dei-dati).

**Comment by agofa** [comment]
@pdonadeo  fatto episodio 45, con questo commento

_Al minuto 40:40 si parla di una nuova mappa di warzone (caldera) che però non trovo, quindi ho messo warzone generico_

**Reply by pdonadeo** [reply]
Segnato, grazie Agostino.

**Comment by sanvisimo** [comment]
Ciao, inizio il 16

**Reply by sanvisimo** [reply]
fatto episodio 16

**Reply by agofa** [reply]
Grazie!

**Reply by pdonadeo** [reply]
> fatto episodio 16

Segnato nel wiki, grazie!

**Reply by sanvisimo** [reply]
minuto 48:27 High score su netflix come consiglio di Bordone

**Comment by sanvisimo** [comment]
Inizio il 15, (anche se avrei voluto il 17, ma almeno non ci sono buchi)

**Reply by sanvisimo** [reply]
Fatto il 15

**Reply by pdonadeo** [reply]
> Fatto il 15

Segnato anche il 15, grazie Simone.

**Reply by sanvisimo** [reply]
aggiungo anche qui
56:36 retrobigini - profilo instagram segnalato da Fossetti

**Comment by agofa** [comment]
Inizio il 44

**Reply by agofa** [reply]
> Inizio il 44

Finito ep 44, @pdonadeo

**Reply by pdonadeo** [reply]
Segnato, grazie.

**Comment by agofa** [comment]
Ho creato il record di ep50
Cover venuta un po' così ....

**Reply by pdonadeo** [reply]
@agofa aspetta un attimo... @zampale tu potresti darci la cover dell'episodio 50?

**Reply by pdonadeo** [reply]
Come non detto, è su Drive. Ora la sostituisco con quella di Zampa.

**Reply by pdonadeo** [reply]
Fatto.

**Comment by agofa** [comment]
Prendo il 43

**Reply by agofa** [reply]
Fatto ep 43 con un commento:
FF14: citano espansione senza però dire il titolo, ho messo quindi il riferimento al gioco
@pdonadeo

**Reply by pdonadeo** [reply]
> Fatto ep 43 con un commento: FF14: citano espansione senza però dire il titolo, ho messo quindi il riferimento al gioco @pdonadeo

Segnato.

**Comment by n1k9** [comment]
Ho iniziato la 17

**Reply by n1k9** [reply]
Ho finito l'ep. 17. Mi riservo di controllare due giochi segnati in scaletta ma che non ho sentito mensionati

**Reply by pdonadeo** [reply]
Segnato anche il 17.

**Comment by agofa** [comment]
Prendo 42

**Reply by agofa** [reply]
@pdonadeo fatto 42 con commenti
26:20 si parla di un'espansione di Outriders, New Horizon, che non ho trovato in rawg
39:48 Serie Netflix, Arcane, dai creatori di League of Legends
41:30 Si parla di due giochi derivati da LoL ma Fossa dice il titolo solo di 1
50:00 Bordone consiglia, con riferimento a Warzone, un video su Vimeo (how to disappear)

**Reply by sanvisimo** [reply]
New horizons dovrebbe essere animal crossing: new horizons

https://rawg.io/games/animal-crossing-2019

**Reply by pdonadeo** [reply]
> New horizons dovrebbe essere animal crossing: new horizons

No, non c'entra niente con Animal Crossing.

**Reply by pdonadeo** [reply]
@agofa ho aggiornato il tuo commento nel DB: effettivamente New Horizons non si trova così ho messo almeno Outriders a 26:20 per non avere un buco di diversi minuti.

**Reply by sanvisimo** [reply]
> > New horizons dovrebbe essere animal crossing: new horizons
> 
> No, non c'entra niente con Animal Crossing.

Scusa, avevo letto male. Hai ragione

**Comment by agofa** [comment]
In lavorazione 41

**Reply by agofa** [reply]
> In lavorazione 41

Fatto 41 con note
Zampa consiglia hw, non ho potuto inserirlo
Bordone disinstalla returnal, comunque l'ho inserito (per Fossa è un consiglio di saggezza)

**Comment by pdonadeo** [comment]
41 e 42 segnati.

**Reply by agofa** [reply]
Quindi dovrebbero mancare, ad ora, 25 episodi

**Comment by agofa** [comment]
Ep 40 in lavorazione

**Reply by agofa** [reply]
> Ep 40 in lavorazione

Fatto il 40 con note

03:30 - si parla di "star ocean the divine force" che non trovo. Ho messo il riferimento ad un titolo citato da Bordone che è l'ultimo della serie prima di questo nuovo
39:56 - di Voice of Cards ce ne sono due, ma il secondo è del 2022 quindi immagino stia parlando del primo
Il primo: https://rawg.io/games/voice-of-cards-the-isle-dragon-roars
Il secondo: https://rawg.io/games/voice-of-cards-the-forsaken-maiden

**Comment by agofa** [comment]
Inizio il 39

**Reply by agofa** [reply]
> Inizio il 39

Fatto 39 con un commento
00:01:56 - Si parla sia di FIFA e di PES in contemporanea per cui li ho messo allo stesso minuto

**Comment by pdonadeo** [comment]
39 e 40 segnati. Direi che è confermato ufficialmente che Agostino è un **EROE**!

**Reply by MarcoJul** [reply]
@agofa quando hai detto che ne mancano 25 non avevamo capito le volessi chiudere entro stasera! 😂

occhio che il frontend va più lento😅

**Reply by agofa** [reply]
Si fa quel che si può :-)

**Comment by agofa** [comment]
Che nessuno si muova! Il 38 è mio.

**Reply by agofa** [reply]
> Che nessuno si muova! Il 38 è mio.

Fatto 38
Segnalo il primo numero di Osservatorio Star Citizen

**Reply by pdonadeo** [reply]
Segnato!

**Reply by pdonadeo** [reply]
Però, caro collega, la devo contraddire: il primo episodio di Star Citizen è il lontano numero 12.

**Reply by agofa** [reply]
> Però, caro collega, la devo contraddire: il primo episodio di Star Citizen è il lontano numero 12.

Che ne so? Loro essi stessi medesimi hanno affermato essere l'episodio inaugurale...

**Reply by zampale** [reply]
Poi controllo, ma può essere che Star Citizen come gioco sia nell'episodio 12, l'osservatorio in quella più avanti

**Reply by pdonadeo** [reply]
È tutto vero, mi sono sbagliato. Modifico l'episodio 12.

**Comment by agofa** [comment]
37 in lavorazione

**Reply by agofa** [reply]
> 37 in lavorazione

37 fatto con note

_21:55 non sono sicuro sia questo il Blood Hunt citato, direi di sì, ma ...
23:30 Si parla di musica
26:10 Un paio di uncharted ma non sono sicuro dei titoli_

**Reply by pdonadeo** [reply]
Segnato!

**Comment by sanvisimo** [comment]
Inizio il 18

**Reply by sanvisimo** [reply]
FInita con note

3:07 - Microsoft acquista Bethesda
10:02 - Game Pass - Xbox Series X/S
14:20 - Console NextGen
24:41 - Luna, Cloud Gaming di Amazon
33:07 - ZAMPA parla di puntata e non episodio
47:56 - bitmap books - game boy boxart collection - Consiglio Zampa
53:20 - Introduzione del termine MORTACCINO

**Comment by agofa** [comment]
inizio 36

**Reply by agofa** [reply]
> inizio 36

Fatto 36 con qualche nota

_25:45 Activision-Blizzard e le molestie (Cosby-Suite)
51:26 citazione di vari giochi solo il titolo (giochi in uscita), ho messo solo quelli di cui hanno parlato un po'_

**Comment by pdonadeo** [comment]
@sanvisimo episodio 18 segnato
@agofa episodio 36 segnato... sulla fiducia 😂

**Reply by agofa** [reply]
Fatto fatto

**Comment by agofa** [comment]
35 in lavorazione

**Reply by agofa** [reply]
> 35 in lavorazione

Fatto 35 con note
_40:30 Playdate
43:28 Steam Deck
48:53 Zampa consiglia un libro: A Guide to Japanese Role-Playing Games_

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
Ep 19 in lavorazione

**Reply by agofa** [reply]
> Ep 19 in lavorazione

Fatto con note

_00:00:10 Si parla di Xbox series X
00:07:00 Si parla di Mortaccini
00:33:20 il merdone di Ubisoft_

Sulla faccenda mortaccini, ho messo il riferimento ai capostipiti
Rogue
Demon Souls
Metroid/Super Metroid - Castelvania

Con questo siamo a 378 giochi citati

**Comment by pdonadeo** [comment]
Segnato anche il 19.

Giuro che questo fine settimana ne riascolto almeno uno o due.

**Comment by bionicco** [comment]
Prendo il 50...che ancora non ho ascoltato, giusto per farvi capire che periodo incasinato sia questo....scusate la poca presenza e colalborazione...ma tanto c'è @agofa  che viaggia come un treno! Mitico!

**Reply by bionicco** [reply]
ep 50 concluso

**Reply by pdonadeo** [reply]
> ep 50 concluso

L'ho aggiunto alla lista e segnato completato.

**Comment by agofa** [comment]
> Giuro che questo fine settimana ne riascolto almeno uno o due.

Lascia stare, tu e @MarcoJul avete altro a cui pensare. Lasciate a noi le trascrizioni

**Comment by agofa** [comment]
Ep 20 iniziato

**Reply by agofa** [reply]
> Ep 20 iniziato

Fatto con note

00:00:00 Prova e recensione PS5
00:32:00 Stadia e Nintendo Cloud con notizia su due giochi per switch usabili solo in cloud

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
Vado di ep21

**Reply by agofa** [reply]
> Vado di ep21

Fatto 21

_Per gran parte del tempo di parla di xBox, ovviamente
31:50 xCloud
46:55 Destiny 2, ho messo il gioco base ma hanno citato espansioni_

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
Ep 22 in lavorazione

**Reply by agofa** [reply]
> Ep 22 in lavorazione

Fatto

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
Inizio 23

**Reply by agofa** [reply]
> Inizio 23

Fatto 23 con note

_00:09:30 Oculus Quest 2
00:41:05 videogioco di Balenciaga (????) da Bordone_

**Reply by pdonadeo** [reply]
Segnato anche il 23.

**Comment by agofa** [comment]
Faccio lo speciale, cioè questo
https://db.euscitojoypad.it/backoffice/episodio/12/change/

Che forse nel(la) wiki compare due volte con titoli diversi, infatti uno dei due punta a un episodio/puntata mancante

**Reply by agofa** [reply]
> Faccio lo speciale, cioè questo

Fatto, in pratica un solo gioco
Ma con qualche nota

_00:01:00 Si inizia parlando delle origini di Nintendo
00:02:30 Game&Watch in generale
00:07:15 Riedizione G&W di Super Mario Bros
00:10:37 Nintendo Labo
00:17:10 Lego Super Mario (che non trovo su rawg)_

**Reply by pdonadeo** [reply]
Segnato completato e ho sistemato il (la) wiki.

**Comment by agofa** [comment]
Prendo il 24 epis...

**Reply by agofa** [reply]
> Prendo il 24 epis...

Fatto.
Abbiamo superato la soglia dei 400 giochi: 405...

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
Lavoro il 34, senza scaletta ...

**Reply by agofa** [reply]
> Lavoro il 34, senza scaletta ...

Fatto
Siamo a 411 giochi ...

**Reply by pdonadeo** [reply]
Segnato.

**Comment by bionicco** [comment]
Fatto il 51 (machimelhafattofare)

**Reply by pdonadeo** [reply]
L'ho aggiunto al (alla?) wiki.

**Reply by bionicco** [reply]
Per questo episodio (ma a questo punto anche per altri passati e futuri, visto che sta diventando un format) ci starebbe bene qualcosa tipo "sinossi bordone" come tipologia di intervento

**Comment by zampale** [comment]
Si guarda, hai proprio pestato il merdone
Il 28 giu 2022, 18:39 +0200, Niccolò Biondi ***@***.***>, ha scritto:

**Comment by agofa** [comment]
> Fatto il 51 (machimelhafattofare)

Io non ho ancora trovato il coraggio di ascoltarlo.
Ci ho paura...

**Comment by agofa** [comment]
ep 33 in lavorazione

**Reply by pdonadeo** [reply]
Segnato... sulla fiducia 😂

**Reply by agofa** [reply]
> Segnato... sulla fiducia 😂

Eh eh eh 
Fatto con note

_00:23:00 Si parla di cosa sarà E3 con raffica di potenziali giochi, nomi citati rapidamente senza molte spiegazioni
00:45:10 Ask Iwata - Libro consigliato da Zampa
00:47:00 Keykron K3  - Tastiera consigliata da Bordone_

**Comment by agofa** [comment]
Inizio il 32

**Reply by agofa** [reply]
> Inizio il 32

Fatto con note


_00:13:00 Si parla di free to play
00:15:20 si parla di Heartland (forse questo: https://register.ubisoft.com/division-experience/en-GB) ma non lo trovo in rawg
Giochi cancellati che non ci sono su rawg
00:22:37 The agent di Rockstar
00:23:42 Deep Down di Capcom

00:42:00 Zampa consiglia Fritt Fighter che è una piattaforma di delivery non è un gioco
00:43:05 Bordone consiglia How to disappear by Total Refusal  che è un cortometraggio_

**Reply by pdonadeo** [reply]
Segnato, grazie!

**Comment by agofa** [comment]
Ora ne mancano sette ...

**Reply by pdonadeo** [reply]
Madonna, ci siamo quasi...

**Comment by agofa** [comment]
Ep 25 in compilazione

**Reply by agofa** [reply]
> Ep 25 in compilazione

Fatto con una nota

_00:36:45 Citano Animal Crossing senza altri dettagli ho messo quello del 2020_

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
31 in lavorazione

**Reply by agofa** [reply]
> 31 in lavorazione

Fatto 31 con note
_00:30:05 si parla di Amnesia che non ho trovato come gioco unico, ho messo quella che dovrebbe essere la collection, ma non garantisco
00:35:00 si parla di Mortaccini e Rinascenti/Muorinasci, Souls like e Rogue like
00:41:00 si parla di E3 e una serie di giochi appena accennati, in uscita, che ho comunque inserito_

**Reply by pdonadeo** [reply]
Segnato anche il 31.

**Comment by sanvisimo** [comment]
Cavolo come siete andati avanti! Complimentoni!

Inizio il 26

**Reply by sanvisimo** [reply]
Finito il 26.
Visto che è praticamente tutto sul torneo infernale dell'epoca vengono citati i giochi a coppie, ad ogni avanzamento ho ripetuto i giochi quando venivano citati.

**Comment by agofa** [comment]
Prendo il 30

**Reply by agofa** [reply]
> Prendo il 30

Fatto con note

_00:34:10 Zampa consiglia un libro: Game Boy: The Box Art Collection
00:37:45 Bordone consiglia un progetto VR IED_

**Comment by agofa** [comment]
Quindi mancano 3 episodi ocomelichiamanoora:
27 28 e 29

E siamo a 492 giochi referenziati (i citati sono un po' di più che alcuni non si trovano su rawg)

**Comment by pdonadeo** [comment]
Segnati 26 e 30.

**Comment by agofa** [comment]
Inizio il 27

**Reply by agofa** [reply]
> Inizio il 27

Fatto con note, poca roba

_00:02:48 Si parla di Reddit e Gamestop e le vendite al ribasso
00:38:00 Viene annunciata MANETTINI, il cui primo numero quindi febbraio 2021
00:44:16 Bordone parla di Wanikani, piattaforma di gamification per imparare gli ideogrammi_

**Reply by pdonadeo** [reply]
Segnato!

**Comment by agofa** [comment]
Meno 2 epicosi e 494 giochi nel db

**Reply by pdonadeo** [reply]
> Meno 2 epicosi e 494 giochi nel db

È incredibile quanti episodi ti sei riascoltato, grazie!

**Comment by zampale** [comment]
Di guarda, eroe. Eroi tutti!
Il 6 lug 2022, 22:38 +0200, Paolo Donadeo ***@***.***>, ha scritto:

**Comment by agofa** [comment]
E via di 29 ...

**Reply by agofa** [reply]
> E via di 29 ...

Fatto anche epirobo 29
Commenti

_00:28:00 Si parla di Nintendo Mario e giochi che non si possono più comprare e del problema della conservazione dei videogiochi
00:38:35 Fantacritic lo spiegone_

**Reply by pdonadeo** [reply]
Segnato… e ne manca **solo uno**!

**Comment by agofa** [comment]
Stasera farò il 28 ...

**Reply by agofa** [reply]
> Stasera farò il 28 ...

Anzi no, fatto ora
_00:01:10 Si parla di state of play
00:28:10 Non sono sicuro sia quel bravely-default-ii (c'è un bravely-default- second)
00:31:45 Anthem Next mai fatto, ma i nostri bravi ne parlano
00:35:58 si parla di Jade Raymond (https://it.wikipedia.org/wiki/Jade_Raymond) che un'autrice
00:40:06 Si parla di patch di Cyberpunk 2077 (l'ho inserito lo stesso)_

**Reply by pdonadeo** [reply]
Segnato.

**Comment by agofa** [comment]
E QUINDI ... F.I.N.I.T.O. fino alla prossima puntata, al prossimo episodio al prossimo coso
CINQUECENTOSEI CITAZIONI...

**Comment by pdonadeo** [comment]
Abbiamo veramente finito! Grazie a tutti!

**Reply by zampale** [reply]
Coff Coff *c'è quello di domani* coff coff.

Comunque non so cosa dire, siete davvero belle persone.

**Reply by pdonadeo** [reply]
@zampale vedi... https://youtu.be/ryF5ymKhn7Q

**Reply by MarcoJul** [reply]
Ora la palla è nella mia metà campo quindi😅 cerco di dare una chiusura durante questo weekend, nel frattempo se avete note sotto con le issue🫠

**Comment by agofa** [comment]
Faccio il 52

**Reply by agofa** [reply]
> Faccio il 52

Fatto con note
_00:15:15 Compleanno Atari, 50 anni
00:19:00 Nel blocco Atari hanno citato vecchi giochi, li ho inseriti per completezza
00:31:00 Si parla del prossimo E3
00:39:55 Si parla di noi (e Bordone dimentica il jingle nello stacco :-) )
00:42:08 Zampa consiglia Nerf + LEGO Bowser
00:44:20 Bordone Sconsiglia un gioco :-)_

**Reply by pdonadeo** [reply]
Segnato, o meglio aggiunto, al wiki.

Posso dire una cosa, sperando che @zampale mi legga? Avevo già sentito della storia assurda riguardante Ron Gilbert e quando sabato ho letto la notifica di Spotify che annunciava _«Quello con la gente che deve stare calma»_ mi sono ribaltato dal ridere.

**Comment by agofa** [comment]
Vado di 53

**Reply by agofa** [reply]
Fatto 53

**Reply by pdonadeo** [reply]
Tra un attimo lo metto on line su staging.

**Comment by agofa** [comment]
Episodio 54 fatto con queste note, che forse la scaletta (quando l'avremo) ci aiuterà a risolvere, @pdonadeo e @zampale 
00:09:47 Zampa cita un gioco che proprio non capisco quale sia, ne parlano per un po'
00:14:36 Fossa cita un gioco di cui non capisco il nome e non lo trovo

**Reply by pdonadeo** [reply]
Allora... come scaletta Zampa mi ha condiviso un foglio Excel e non la solita cartellina divisa per episodio. Questo file è enigmatico: contiene solo la descrizione dell'episodio e due link all'episodio 36... Cosa avrà voluto dirci Zampa?

![JF-300x225](https://user-images.githubusercontent.com/242049/189502013-dbabf51f-06b9-4dfa-85db-644ec80c3526.jpg)

**Reply by agofa** [reply]
GOMBLOTTO!11

**Comment by zampale** [comment]
Dead Island 2
High on Life

Resto, tutto condiviso!
Il 10 set 2022, 19:20 +0200, Agostino Fanti ***@***.***>, ha scritto:

**Reply by pdonadeo** [reply]
Grazie @zampale 😃 

Per tutti: Zampa mi ha mandato la scaletta, la trovate nel solito posto nella cartella "Stagione 4".

**Reply by agofa** [reply]
Grazie, un grande aiuto per noi che sentiamo poco e capiamo meno. 🤪

**Reply by agofa** [reply]
Ho sistemato l'epicoso inserendo le due citazioni

**Comment by agofa** [comment]
@zampale appena puoi la scaletta su Gogol Drive, che se non mi addormenterò alle 20:59:59 domani sera "sbobinerò" la puntat, l'episod, insomma, la cosa di stasera, numero LV

**Reply by pdonadeo** [reply]
@agofa Colpa mia che non l'ho copiata, attimo che arriva...

**Reply by pdonadeo** [reply]
@agofa è on line nella solita cartellina Drive.

**Reply by agofa** [reply]
Grazie. Anche se, comunque, si deve aspettare la versione "podcast"

**Reply by agofa** [reply]
Ok domani sabato, penso nel pomeriggio, trascriverò la puntata.
L'episodio
Insomma quella cosa lì

**Reply by pdonadeo** [reply]
> Ok domani sabato, penso nel pomeriggio, trascriverò la puntata. L'episodio Insomma quella cosa lì

Intanto ho aggiunto la puntata al database, completa di tutto (link, cover, descrizione).

**Reply by agofa** [reply]
Fatto con qualche nota
00:07:00 Fossa parla di Playstation VR 2
00:14:05 Fossa cita ENOTRIA - Project Galileo che non trovo su rawg
00:30:00  I leak di GTA VI
00:42:30 Parlano di noi e di Joycord
00:47:45 Zampa consiglia la Comic Edition di Batman Arkham City
00:49:05 Bordone consiglia il trailer della serie HBO The Last of Us
00:50:50 Fossa consiglia un anime basato su Cyberpunk: Edgerunners

**Comment by agofa** [comment]
@pdonadeo appena possibile dovresti copiare la scaletta del 56
L'immagine che @zampale ha messo su discord è scomoda da usare

Grassssie

**Reply by pdonadeo** [reply]
> @pdonadeo appena possibile dovresti copiare la scaletta del 56 L'immagine che @zampale ha messo su discord è scomoda da usare

È tutto su Drive. Ho già aggiunto l'episodio ma c'è un ma...

![immagine](https://user-images.githubusercontent.com/242049/197235740-9a5e2b6e-9bfd-43e5-a642-f92f7163bd49.png)

Che è un po' "spoiler". Lo tolgo?

**Reply by agofa** [reply]
epicoso 56 fatto
Le note
00:00:00 Zampa, la tosse e le caramelle di Bordone
00:06:00 Si parla di sfavillotti
00:35:05 La fine di Stadia
00:40:40 E' uscito joypad
00:42:40 Joycord
00:44:35 Fantacritic su Joycord

**Comment by agofa** [comment]
Ah boh?
Direi di togliere
Un flag "nascondi episodio" nel retrobottega?
Utile anche in caso di manutenzione del singolo episodio

(vabbé, vado vado)

**Reply by pdonadeo** [reply]
Sì, assolutamente. Ormai stiamo diventando un redazione (somewhat...) e ci serve quel flag.

Mi apri una issue? Non ho tempo adesso.
